Tests/PostgresNIOTests/New/Data/JSON+PSQLCodableTests.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 03.02.21.
+//
+
+import XCTest
+@testable import PostgresNIO
+
+class JSON_PSQLCodableTests: XCTestCase {
+    
```

Comment:
I hope you plan to test some more complex JSON at some point 🙂

Reply:
Well, we don't need to test the JSONEncoder and JSONDecoder here, do we? We only need to test that they are indeed invoked...